llvm/include/llvm/TableGen/Record.h

```diff
@@ -120,9 +121,7 @@ class BitRecTy : public RecTy {
   }
 
   static const BitRecTy *get(RecordKeeper &RK);
-
```

Comment:
Whitespace removal is controversial.

Reply:
Right :( I tried to remove whitespace between a series of single-line declaration/defs with no comments or anything else in the middle. But let's see during review, I can revert back if there is pushback,

Reply:
Or should I interpret your comment as that pushback and do it proactively?

Reply:
I think it should be reverted. It is not standardized and personal opinions might differ, so someone after you could try to put whitespaces back because they find it more readable.
The general rule of thumb is not to reformat nearby code unless it makes it conformant to the coding style.

Reply:
Sounds good, will revert the whitespace elimination

Reply:
Done. Resolving